staging/src/github.com/openshift/sdn/cmd/sdn-cni-plugin/openshift-sdn_linux.go

```diff
@@ -194,17 +198,21 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 			return fmt.Errorf("failed to configure container loopback: %v", err)
 		}
 
+		var dsts []*net.IPNet
 		// Set up macvlan0 (if it exists)
 		link, err = netlink.LinkByName("macvlan0")
 		if err == nil {
+			var addrs []netlink.Addr
 			err = netlink.LinkSetUp(link)
 			if err != nil {
 				return fmt.Errorf("failed to enable macvlan device: %v", err)
 			}
+			for _, addr := range addrs {
+				dsts = append(dsts, &net.IPNet{IP: addr.IP, Mask: net.CIDRMask(32, 32)})
```

Comment:
er, you can't move this here; `addrs` hasn't been set yet. Just leave the `addrs` code where it was